static/templates/settings/admin_settings_modals.hbs

```diff
@@ -0,0 +1,7 @@
+{{> deactivation_user_modal }}
+
+{{> realm_domains_modal }}
+
+{{> resend_invite_modal }}
+
+<div id="user-info-form-modal-container"></div>
```

Comment:
These are probably all candidates to be converted to the `confirm_dialog` API as a follow-up PR.

Reply:
Cool! Will work on refactoring them to use the `confirm_dialog`.